tests/test_download.py

```diff
@@ -47,10 +49,18 @@ def evr(self):
 
     @property
     def sourcerpm(self):
+        name = self.name
+
+        # special cases for debuginfo tests
```

Comment:
could source pkg name be defined by optional param in constructor, please? (If not defined than it would be same as pkg name)